integration-tests/tests/link-to-new-workspace.js

```diff
@@ -0,0 +1,42 @@
+const { checkbox, click, clickable, clickTableCell, fillIn, waitForNoSpinners } = require('../utils/integration-utils')
+const { enableDataCatalog, testWorkspaceName } = require('../utils/integration-helpers')
+const { withUserToken } = require('../utils/terra-sa-utils')
+
+
+const testLinkToNewWorkspaceFn = withUserToken(async ({ testUrl, page, token }) => {
+  await enableDataCatalog(page, testUrl, token)
+  await click(page, clickable({ textContains: 'browse & explore' }))
+  await waitForNoSpinners(page)
+
+  await click(page, checkbox({ text: 'Granted', isDescendant: true }))
+  await clickTableCell(page, "dataset list", 2, 2)
+  await waitForNoSpinners(page)
+  await click(page, clickable({ textContains: 'Link to a workspace' }))
+  await waitForNoSpinners(page)
```

Comment:
`run-catalog-workflow` test also has these 9 lines, which makes me wonder if it'd make sense to factor them out.